app/src/main/java/no/usn/mob3000/ui/components/settings/SettingsSectionApplication.kt

```diff
@@ -84,14 +86,22 @@ fun SettingsSectionApplication(
             onDismissRequest = { languageExpanded = false }
         ) {
             DropdownMenuItem(
-                text = { Text(stringResource(R.string.settings_section_app_language_enus)) },
+                text =
+                { Text(
```

Comment:
Incorrect indentation. The way to indent these has been shown before. You do not indent the starting bracket, you indent the composable function inside the unit body. It's important to follow the Kotlin style guide. This should be as below.
```kotlin
DropdownMenuItem(
    text = {
        Text(
            stringResource(R.string.settings_section_app_language_enus),
            color = MaterialTheme.colorScheme.onBackground)
    },
```